packages/core/__tests__/interpreter/onerror.js

```diff
@@ -1,10 +1,17 @@
+/* eslint-disable no-console */
 import { createStore, applyMiddleware } from 'redux'
 import sagaMiddleware from '../../src'
 import * as io from '../../src/effects'
+
 test('saga onError is optional', () => {
+  let consoleError = console.error
```

Comment:
shouldn't it be a separate test? doesn't look like `saga onError is optional`:)

Reply:
that's also kinda true - this relation is implicit, maybe we could just add a note to the `test`'s description, smth like:
```js
test('saga onError is optional (the default is console.error)', () => {
  // ...
})
```

Reply:
👍